doc/product-reference.md

```diff
@@ -142,7 +142,7 @@ metadata:
   name: my-secret
   namespace: 3scale-test
 data:
-  issuerEndpoint: https://3scale-zync:some-secret@keycloak-rhsso-test.apps.xxxxx.xxxx.s1.devshift.org/auth/realms/petstore
+  issuerEndpoint: https://3scale-zync:secret-name@keycloak-rhsso-test.example.com/auth/realms/petstore
```

Comment:
`secret-name`? that can create confusion

Reply:
could you please suggest how to write this. Thank you!

Reply:
`some-secret` is ok for me

Reply:
Yes, sure, it was confusion, mistake, in previous revision it was `some-secret`. Thank you very much!  Updated.